cmds/core/shasum/shasum.go

```diff
@@ -17,26 +17,37 @@ import (
 	"os"
 )
 
-var algorithm int
+type Algorithm int
 
-var usage = "Usage:\nshasum -a <algorithm> <File Name>"
+const (
+	Sha1   Algorithm = 1
```

Comment:
I think it should be SHA1

like here https://cs.opensource.google/go/go/+/refs/tags/go1.23.1:src/crypto/sha512/sha512.go:
```go
crypto.RegisterHash(crypto.SHA512, New)
```

Reply:
Sounds good, will make that all caps.